Assignment_display_marks/Student.php

```diff
@@ -0,0 +1,107 @@
+<?php
+  /**
+   * Student class to store student operations
+   */
+  class Student
+  {
+    public $student;
+    public $subject;
+
+    function __construct($student,$subject)
+    {
+      $this->student = $student;
+      $this->subject = $subject;
+    }
+    public function get_data()
+    {
+      // $this->show_details($this->student);
+      // $this->show_details($this->subject);
+    }
+    public function subject_fetch($grade){
```

Comment:
Do one thing, return the subjects array from this method and display it on index.php.